initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
As Mike mentioned the rpm erase removes /opt/HPCCSystems even if it is a link and to change that is an arduous task to alter the spec file contents, affecting install as well.  The other install dirs can be handled with this logic to test for soft link.